portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,52 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+/** 
+ * Toggles the drop down text.
+ */
+function toggleDropDown(id) {
+  document.getElementById(id).classList.toggle("show");
+}
+
+/**
+ * Adds a random favorite quote to the page.
+ */
+function addRandomFavoriteQuote() {
+  const favoriteQuotes =
+      ["Many years later, as he faced the firing squad, Colonel Aureliano Buendía was to remember that distant afternoon when his father took him to discover ice. \n —Gabriel García Márquez, One Hundred Years of Solitude",
+      "Hope is a dangerous thing. Hope can drive a man insane. \n - The Shawshank Redemption",
+      "Bran thought about it. 'Can a man still be brave if he's afraid?' 'That is the only time a man can be brave,' his father told him. \n ― George R.R. Martin, A Game of Thrones"];
+
+  // Picks a random quote.
+  const randomFavoriteQuote = favoriteQuotes[Math.floor(Math.random() * favoriteQuotes.length)];
+
+  // Adds the favorite quote to the page.
+  const favoriteQuoteContainer = document.getElementById('favorite-quote-container');
+  favoriteQuoteContainer.innerText = randomFavoriteQuote;
+}
+
 /**
- * Adds a random greeting to the page.
+ * Adds a random fun fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFunFact() {
+  const funFacts =
+      ["sister", 
+      "selfie",
+      "I'm a fan of collections. I collect postcards, stickers, books, snacks, etc.",
+      "My English name is Heidi. Many people can't pronounce that.",
+      "Funny enough that most of my Chinese friends call me by my English name while my American friends call me by my Chinese name."];
```

Comment:
Follow this pattern to define arrays: https://google.github.io/styleguide/jsguide.html#features-array-literals